From de58c695fafb97061cbeef4ecdd2af105b7eccaf Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Sat, 17 Oct 2015 13:02:36 +0200
Subject: [PATCH 06/14] Revert "Fully qualify libEGL.so.1, libEGLESv2.so.2
 libraries"

This reverts commit d07fba149ddc6df4cbc99d31b1edb17331fcfe53.
For some obscure reason, having this method fail on our devices
improves the compatibility with our EGL libraries.
Some more investigation is needed on that.
---
 src/core/surface_factory_qt.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/core/surface_factory_qt.cpp b/src/core/surface_factory_qt.cpp
index 97a3047..11d8f3b 100644
--- a/src/core/surface_factory_qt.cpp
+++ b/src/core/surface_factory_qt.cpp
@@ -74,13 +74,13 @@ base::NativeLibrary LoadLibrary(const base::FilePath& filename) {
 bool SurfaceFactoryQt::LoadEGLGLES2Bindings(AddGLLibraryCallback add_gl_library, SetGLGetProcAddressProcCallback set_gl_get_proc_address)
 {
     base::FilePath libEGLPath = QtWebEngineCore::toFilePath(QT_LIBDIR_EGL);
-    libEGLPath = libEGLPath.Append("libEGL.so.1");
+    libEGLPath = libEGLPath.Append("libEGL.so");
     base::NativeLibrary eglLibrary = LoadLibrary(libEGLPath);
     if (!eglLibrary)
         return false;
 
     base::FilePath libGLES2Path = QtWebEngineCore::toFilePath(QT_LIBDIR_GLES2);
-    libGLES2Path = libGLES2Path.Append("libGLESv2.so.2");
+    libGLES2Path = libGLES2Path.Append("libGLESv2.so");
     base::NativeLibrary gles2Library = LoadLibrary(libGLES2Path);
     if (!gles2Library)
         return false;
-- 
2.7.4

